Modules/OrgUnit/classes/Exceptions/ilPositionPermissionsNotActive.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+require_once("./Modules/OrgUnit/classes/Exceptions/class.ilOrguException.php");
+
+/**
+ * Class ilPositionPermissionsNotActive
+ *
+ * @author Oskar Truffer <ot@studer-raimann.ch>
+ */
+class ilPositionPermissionsNotActive extends ilOrguException {
+	/** @var string  */
+	protected $object_type = "";
+
+	/**
+	 * ilPositionPermissionsNotActive constructor.
+	 *
+	 * @param string $a_message
+	 * @param string $type
+	 * @param int    $a_code
+	 */
+	public function __construct($a_message, $type, $a_code = 0) {
```

Comment:
I would recommend to avoid the `a_` prefix, because it has actually no informative value.

Reply:
while i agree, it stands in conflict with naming the variables the same as in the superclass.